### ISSUE TITLE
Follow up of #19818 - Downloading multiple torrent files in a single browser session fails to download the torrent files.

### ISSUE THREAD

**Issue by GeetaSarvadnya** [brave/brave-browser#20298]
## Description 

Found while testing https://github.com/brave/brave-browser/issues/19818
Downloading multiple torrent files in a single browser session fails to download the torrent files.

## Steps to Reproduce

1. Clean profile 1.34.77
2. Open https://webtorrent.io/free-torrents
3. Click on the torrent/magnet link in `Cosmos Laundromat (torrent file) (magnet link)`
4. Click on `Start Torrent`
5. Torrent files loads and works as expected
6. Click on torrent/magnet link in `Big Buck Bunny (torrent file) (magnet link)`
7. Click on `Start torrent`
8. Torrent file doesn't load

## Actual result:

Downloading multiple torrent files in a single browser session fails to download the torrent files.
## Expected result:

Able to download multiple torrent files at a time in a single browser session

## Reproduces how often: 

Easy
## Brave version (brave://version info)

<html>
<body>

Brave | 1.34.77 Chromium: 97.0.4692.56 (Official Build) (64-bit)
-- | --
Revision | 04da6c66398ca50e603cc236a07dc7dfd3bbc750-refs/branch-heads/4692@{#990}
OS | Windows 10 Version 21H2 (Build 19044.1415)

</body>
</html>

## Version/Channel Information:

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel? Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @spylogsster @brave/legacy_qa @rebron

**Comment by rebron** [comment]
cc: @yrliou Can you take a look?

**Comment by Kromasm** [comment]
Hi !

Any news on this bug ? I'm experiencing the same problems (Brave 1.38.111). Thank you for your work ! :)

**Comment by mihaiplesa** [comment]
In the meanwhile one workaround is to disable WebTorrent in Brave while this gets fixed.

**Comment by thiagomgd** [comment]
Same happening here: 
Version 1.44.105
MacOs 12.6

**Comment by saurabhgupta050890** [comment]
Same issue

**Comment by Mikeatalol** [comment]
Kinda bs if you ask me, still happening in 2023 on Version 1.51.110 Chromium: 113.0.5672.77 (Official Build) (64-bit) wich i JUST updated to...

**Comment by harrymmmm** [comment]
and here we are in 2024/7 ...
This worked well several years agfo for me in Brave-beta (on ubuntu), but then even the first torrent failed every time. It hasn't worked since then, multiple OS version, multiple Brave-beta versions, numerous reboots etc ...

**Comment by HandsomeTopKhi** [comment]
I don't think this will ever be solved. I'm just going to stop using Brave due to this issue.

**Comment by Vahtera** [comment]
Almost three years and this same issue still persists? What is going on?

**Comment by saqketram** [comment]
Solution1:
Reset Brave’s Built-in Torrent Client:
=============================
To resolve this issue, follow these steps:
Open Brave and go to:
brave://settings/?search=torrent
Locate the setting "Brave’s built-in torrent client."
Disable the setting and then re-enable it.
If it didn't work, once Restart Brave.

After performing these steps, Brave’s WebTorrent client should properly download multiple torrents again.

Solution2:
Clear browser cache & site data:
==========================
Open brave://settings/clearBrowserData
Select "Cookies and other site data" and "Cached images and files."
Click "Clear data" and restart Brave.

Solution3:
Try a different browser profile:
=========================
Open brave://settings/people
Create a new profile and test torrents there.

This issue was opened 3 years ago and this bug is still not fixed by brave. I wish I could help in fixing this bug on Braves end.

**Comment by bsclifton** [comment]
Closing as `wontfix`.

WebTorrent as a feature has been removed with https://github.com/brave/brave-browser/issues/46416 which goes out with version 1.81 (releasing 5th of August, 2025)

See https://github.com/brave/brave-browser/issues/46430 for more information
